packages/core/src/components/MobileCarousel/MobileCarousel.js

```diff
@@ -1,17 +1,17 @@
 import PropTypes from 'prop-types';
 import React, { Children } from 'react';
-import shortid from 'short-id';
-import SectionHeader from '../Type/SectionHeader/SectionHeader';
+import nanoid from 'nanoid';
 
 import css from './MobileCarousel.css';
 
 const MobileCarousel = ({ children = [], title }) => {
+  var id = nanoid();
```

Comment:
the id needs to be generated in the loop for each element

Reply:
🤦‍♂ totally missed this. good spot!